docs/generate/readme.md

```diff
@@ -351,8 +352,14 @@ Only Python supports generating
 |`--no-namespace-folders`|Specify if you don't want pkgutil-style namespace folders. See [here](https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages) for more information on pkgutil namespace folders. Defaults to `false`.| x | | | | | |
 |`--credential-default-policy-type=BearerTokenCredentialPolicy|AzureKeyCredentialPolicy`|Specify the default credential policy (authentication policy) for your client. Use in conjunction with `--add-credential`. Currently only supports [`BearerTokenCredentialPolicy`](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-core/latest/azure.core.pipeline.policies.html#azure.core.pipeline.policies.BearerTokenCredentialPolicy) and [`AzureKeyCredentialPolicy`](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-core/latest/azure.core.pipeline.policies.html#azure.core.pipeline.policies.AzureKeyCredentialPolicy). Default value is `BearerTokenCredentialPolicy`. `--credential-scopes` is tied with `BearerTokenCredentialPolicy`, do not pass them in if you want `AzureKeyCredentialPolicy`.| x | | | | | |
 |`--credential-key-header-name=NAME`|The name of the header which will pass the credential. Use if you have `--credential-default-policy-type` set to [`AzureKeyCredentialPolicy`](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-core/latest/azure.core.pipeline.policies.html#azure.core.pipeline.policies.AzureKeyCredentialPolicy). For example, if generating cognitive services code, you might use `--credential-key-header-name=Ocp-Apim-Subscription-Key`.| x | | | | | |
+|`--library-name=NAME`|The name of your library. This is what will be displayed on NuGet.| | x| | | | |
+|`--shared-source-folders=VALUE(S)`|Pass shared folder paths through here. Common values point to the [shared generator assets](https://github.com/Azure/autorest.csharp/tree/feature/v3/src/assets/Generator.Shared) and [shared azure core assets](https://github.com/Azure/autorest.csharp/tree/feature/v3/src/assets/Azure.Core.Shared) in [autorest.csharp](https://github.com/Azure/autorest.csharp)| | x| | | | |
+|`--save-inputs`|Whether to save the configuration files (i.e. `Configuration.json` or `codeyaml.json`). Defaults to `false`.| | x| | | | |
```

Comment:
This is a flag used mostly for debugging. Not sure if it needs mentioning here.

Reply:
Sounds good, i'll remove it from this list, will talk about it in debugging section instead